lib/VerifiableCredentialStore.js

```diff
@@ -40,28 +40,30 @@ export default class VerifiableCredentialStore {
   /**
    * Gets all verifiable credential instances that match the given parameters
    *
-   * @param {string} [type]
-   * @param {string} [issuer]
-   * @param {boolean} [displayable]
+   * @param {object|Array} query
    *
    * @return {Promise<Array>} List of matching VCs
    */
-  async find({type, issuer, displayable}) {
-    const equals = {};
-    if(type) {
-      if(Array.isArray(type)) {
-        // needs to be implemented on the edv-client side first
-        throw new Error('"type" as array is not implemented.');
-      } else {
-        equals['content.type'] = type;
-      }
-    }
-    if(issuer) {
-      equals['meta.issuer'] = issuer;
+  async find({query} = {}) {
+    if(!query) {
+      throw new Error('"query" MUST be defined.');
```

Comment:
No need to yell.  :)
```suggestion
      throw new TypeError('"query" is a required parameter.');
```